src/OpenTelemetry/Trace/Export/SimpleSpanProcessor.cs

```diff
@@ -28,6 +30,7 @@ public class SimpleSpanProcessor : SpanProcessor
         private readonly SpanExporter exporter;
         private bool disposed = false;
         private long spanCount = 0;
+        private List<SpanData> batch = new List<SpanData>();
```

Comment:
instead of ever-growing list that may cause Out of Memory exception - please track the metrics for the last minute and last hour